src/SessionManager/ProgramSettings.cc

```diff
@@ -75,8 +75,9 @@ ProgramSettings::ProgramSettings(int argc, char** argv) {
  2   error
  3   warning
  4   info
- 5   debug
- 6   trace)");
+ 5   info + performance messages
+ 6   debug
+ 7   debug + protocol messages)");
```

Comment:
We should restore this to just `5 debug`, and use explicit optional flags called `log-performance` and `log-protocol-messages` to turn these options on.

If you add the flags, I will update the help messages.

Reply:
So there are 5 verbosity levels, and we add `--log-performance` and `--log-protocol-messages` options?

Reply:
Yes, that should work. Although we should be consistent with our use of _ in our arguments

Reply:
Yes, that's right. But only 5 verbosity levels (excluding `off`) -- we currently aren't actually using `trace` for anything, so we should remove it completely.

Reply:
My bad -- as @veggiesaurus points out, we use underscores in our option names, so we should use `log_performance` and `log_protocol_messages`.

Reply:
OK. Will do it later.